src/riak_kv_ts_svc.erl

```diff
@@ -675,6 +675,8 @@ make_table_created_missing_resp(Table) ->
       ?E_CREATED_GHOST,
       flat_format("Table ~s has been created but found missing", [Table])).
 
+to_string(X) when is_atom(X) ->
```

Comment:
Necessary, no, but performance-related, so IMHO such a simple optimization should be employed.

Yes, the result in terms of testable value is equivalent. `io_lib:format/2` needs to parse the format string which is surely more costly than simply calling `atom_to_list` when the type is known to be atom. At its core, io_lib:format/2 assumedly does an equivalent to atom_to_list/1, so all of the code between the pattern match on `is_atom/1` to `atom_to_list` is waste. Additionally, needing to unpack the result using `lists:flatten/1` is a waste. These wastes should be used for the general cases that we don't explicitly know about (whether we should have a general case here is questionable since this is an internal function where we can/should know every type that we are converting).